src/index.html

```diff
@@ -13,20 +16,33 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      action="#"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="form__searchBar form__searchBar--big"
```

Comment:
Avoid using multiple classes that modify the same element in a way that their purpose overlaps, such as `form__searchBar` and `form__searchBar--big`. If `--big` is a modifier, ensure it only adds modifications to `form__searchBar` rather than redefining styles.